### ISSUE TITLE
Positive lookahead

### ISSUE THREAD

**Issue by rafaeldelboni** [semilin/aoc23#1]
I was searching for a way to use positive lookahead on cl-ppcre and I found this repo with this comment
https://github.com/semilin/aoc23/blob/main/d1/main.lisp#L36-L39

Have you had the same problem I listed in the issue?
https://github.com/edicl/cl-ppcre/issues/54

_Ps. This is super frustrating because I decided to learn CL in this year Advent of Code and in my first attempt I spend an entire day on this regex hahaha_

**Comment by semilin** [comment]
Yep haha, I had the exact same issue. Spent way too much time trying to get it to work, but I think you're right that it's an issue with cl-ppcre.
